packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -294,6 +301,8 @@ export class GarbageCollector implements IGarbageCollector {
         private readonly gcOptions: IGCRuntimeOptions,
         /** After GC has run, called to delete objects in the runtime whose routes are unused. */
         private readonly deleteUnusedRoutes: (unusedRoutes: string[]) => void,
+        /** For a given node id, returns its package path. */
+        private readonly getNodePackagePath: (nodeId: string) => readonly string[] | undefined,
```

Comment:
Can this be forward looking and use paths, not IDs? To support unified channels?
I'm afraid on size of baggage we are adding that will make it hard to do this work.

Reply:
Yes, sure. The `DataStore` will have to convert it to an id since that's what it understands.